test/manual/mongodb_oidc_azure.prose.test.ts

```diff
@@ -0,0 +1,202 @@
+import { expect } from 'chai';
+
+import {
+  Collection,
+  CommandFailedEvent,
+  CommandStartedEvent,
+  CommandSucceededEvent,
+  MongoClient,
+  OIDC_WORKFLOWS
+} from '../mongodb';
+
+describe('OIDC Auth Spec Prose Tests', function () {
+  const callbackCache = OIDC_WORKFLOWS.get('callback').cache;
+  const azureCache = OIDC_WORKFLOWS.get('azure').cache;
+
+  describe('3. Azure Automatic Auth', function () {
+    let client: MongoClient;
+    let collection: Collection;
+
+    afterEach(async function () {
+      await client?.close();
+    });
+
+    describe('3.1 Connect', function () {
+      before(function () {
+        client = new MongoClient(process.env.MONGODB_URI);
```

Comment:
Since we're using 'mocha_mongodb.json' as the mocha config we should be able to use this.configuration.newClient here

Reply:
Update all instances to use `this.configuration.newClient()` now.